code/TemplateStudioForWinUICs/Templates/Pg/WebView/ViewModels/Param_ItemNameViewModel.cs

```diff
@@ -33,48 +32,42 @@ public bool IsLoading
         set => SetProperty(ref _isLoading, value);
     }
 
-    public bool HasFailures
-    {
-        get => _hasFailures;
-        set => SetProperty(ref _hasFailures, value);
-    }
+    [ObservableProperty]
+    private bool hasFailures;
 
-    public ICommand BrowserBackCommand
+    public Param_ItemNameViewModel(IWebViewService webViewService)
     {
-        get;
+        WebViewService = webViewService;
     }
-
-    public ICommand BrowserForwardCommand
+    [RelayCommand]
+    private async Task OpenInBrowser()
     {
-        get;
+        if (WebViewService.Source != null)
+        {
+            await Windows.System.Launcher.LaunchUriAsync(WebViewService.Source);
+        }
     }
-
-    public ICommand ReloadCommand
+    [RelayCommand]
+    private void Reload()
     {
-        get;
+        WebViewService.Reload();
     }
-
-    public ICommand RetryCommand
+    [RelayCommand]
```

Comment:
Add a CanExecute method (for `CanGoForward`) and add as a parameter here. This will also allow for the enabling/disabling of the button bound to this command. In turn, this will provide a visual indicator of if the button will do anything when clicked.